requirements.txt

```diff
@@ -0,0 +1,149 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Manually? How does this work with renovate?

Reply:
We don't use renovate, we use dependabot.
We used `pip-compile` to first freeze the dependencies and from there it's up for dependabot to make the changes.

Reply:
Understood